screeps-game-api/src/objects/impls/room_position.rs

```diff
@@ -73,6 +88,23 @@ impl RoomPosition {
         ))
     }
 
+    pub fn find_path_to<F, T>(&self, target: &T, opts: FindOptions<F>) -> Vec<Step> 
+    where
+        F: Fn(String, CostMatrix) -> (),
```

Comment:
I think this return type needs to be changed to match `find_path`'s `F`'s return type.